command/job_inspect.go

```diff
@@ -162,7 +162,11 @@ func (c *JobInspectCommand) Run(args []string) int {
 	}
 
 	// Print the contents of the job
-	req := api.RegisterJobRequest{Job: job}
```

Comment:
need a class for deserializing a `Job: `-embedded job spec, without all of the other job registration options.